core-ui/src/components/Extensibility/components-form/ResourceRefRenderer.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { ExternalResourceRef } from 'shared/components/ResourceRef/ExternalResourceRef';
+import { useGetList } from 'shared/hooks/BackendAPI/useGet';
+import pluralize from 'pluralize';
+
+export function ResourceRefRender({
+  onChange,
+  onKeyDown,
+  value,
+  schema,
+  storeKeys,
+  required,
+  ...props
+}) {
+  const resourceType = pluralize(schema.get('kind') || '').toLowerCase();
+
+  const url = `/api/v1/${resourceType}`;
+
+  const { data, loading } = useGetList()(url);
```

Comment:
I believe it would be good to add error handling. We could pass the `error` from `useGetList` to `ExternalResourceRef` and display it there.